horiz_interp/horiz_interp.F90

```diff
@@ -231,10 +231,11 @@ subroutine horiz_interp_init
      write (unit, nml=horiz_interp_nml)
   endif
 
-  if( reproduce_siena ) then
-     call mpp_error(FATAL, "horiz_interp_mod: You have overridden the default value of reproduce_siena " // &
-                           "and set it to .true. in horiz_interp_nml. This is a temporary workaround to " // &
-                           "allow for consistency in continuing experiments. Please remove this namelist " )
+  if (reproduce_siena) then
+     if (mpp_pe() == mpp_root_pe()) call mpp_error(FATAL, "horiz_interp_mod: You have overridden the default " // &
```

Comment:
why is only `mpp_root_pe` calling the fatal?

Reply:
@thomas-robinson, that's my fault.  The error messages were being written out by all the pe's

Reply:
@mlee03 Yes printing the error is good for a `FATAL` message.  All of the PEs should hit this, call `MPI_ABORT` in `mpp_error`, and produce the same tracebacks.  If some of them go past this, it will create different tracebacks for different cores even though they all experienced the same issue. 

The only time you want to potentially limit a call to mpp_error would be if you are calling a `WARNING`, especially in a loop.

Reply:
Thanks for catching this, now that I have a better understanding I will go ahead and remove the ```if (mpp_pe() == mpp_root_pe())``` preceding all of the FATAL errors.